ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/activity-picklist-remote-auto-complete-options.component.ts

```diff
@@ -39,7 +39,9 @@ interface CategorizedPicklistOptions {
             [displayWith]="displayAutoComplete"
         >
             <mat-option
-                *ngFor="let option of picklistOptions.currentAutoCompleteOptions"
+                *ngFor="
```

Comment:
Do you use an auto-format ? 
This change does not improve anything.

Reply:
Yes I do. It doesn't. I got an error for max length (eslint error) so I wasn't able to deploy